.golangci.yml

```diff
@@ -22,6 +22,8 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
+  nolintlint:
+    allow-unused: false
```

Comment:
Why do we need this one?
What it does?

Reply:
it checks for unnecessary `//nolint` comments

for example a line was to long, so we added `//nolint:lll` to it

in another moment, the line was changed, and it is now not too long, so `//nolint` comment is not necessary anymore

with this config, that line would be highlighted to remove the now not needed `//nolint` comment